glide.yaml

```diff
@@ -114,13 +114,13 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: release-4.4
+  version: 3e8f22fb0b56808d981b0a98cd2741d31368aec5
 - package: github.com/openshift/client-go
   version: release-4.4
 - package: github.com/openshift/library-go
-  version: release-4.4
+  version: 993204ef63fd55794c4782936c0d369b22b1e6a1
 - package: github.com/openshift/apiserver-library-go
-  version: release-4.4
+  version: ba8b80421765a458f9c50f767bbb5013b5bd5fdc
```

Comment:
We should keep the branch names. Glide has this nice feature of symbolic versions. It makes the next bump easy.

Reply:
sure, np.

Reply:
done, slightly bigger diff `15 files` vs `20 files` - not bad.